pkg/deploy/server/server_configmap.go

```diff
@@ -242,7 +242,7 @@ func (s *CheServerReconciler) getCheConfigMapData(ctx *chetypes.DeployContext) (
 
 	s.updateUserClusterRoles(ctx, cheEnv)
 
-	for _, oauthProvider := range []string{"bitbucket", "gitlab", constants.AzureDevOpsOAuth} {
+	for _, oauthProvider := range []string{"bitbucket", constants.AzureDevOpsOAuth} {
```

Comment:
Do you happen to know if che-serve still uses `CHE_INTEGRATION_GITLAB_SERVER__ENDPOINTS` env var?

Reply:
I removed the env var usage in the che-server pull request: https://github.com/eclipse-che/che-server/pull/731/files#diff-deca31b9cca47d5b0356dd75fc0fd0c8cb980f3aa029da27b475bdbb37353eb0L650-L651